### PR TITLE
Adds the MRDT enketo widget and android integration

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

This adds a new widget for MRDT verification which launches a third party app which
returns a photo of the MRDT result which gets attached to the report and displayed in
the report view.

Attaching binary content to the report is done generically so other widgets can use the
same pattern.

medic/medic-webapp#4743
medic/medic-webapp#4745
medic/medic-webapp#4742

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: demo-forms/mrdt-demo.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <h:head>
+    <h:title>MRDT Demo</h:title>
+    <model>
+      <instance>
+        <mrdt delimiter="#" id="mrdt_demo">
```
Comment (dianabarsan) [inline demo-forms/mrdt-demo.xml]: Without editing the `id` from `mrdt_demo` to `mrdt`, I couldn't get the image to show up on report detail. Not sure if this is common knowledge or if this form is supposed to be `plug-and-play` functional or not.

File: webapp/src/js/enketo/widgets/mrdt.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            }
+        } );
+    };
+
+    module.exports = {
+        'name': pluginName,
+        'selector': '.or-appearance-mrdt-verify',
+    };
+} );
+
```
Comment (dianabarsan) [inline webapp/src/js/enketo/widgets/mrdt.js]: It appears that this file is indented with 4 spaces instead of 2.

Comment (garethbowen) [reply webapp/src/js/enketo/widgets/mrdt.js]: Yeah... for some reason all the enketo widgets are which keeps them consistent with the enketo framework. I'm not sure why but some people passionately believe that's important.

Comment (garethbowen) [reply webapp/src/js/enketo/widgets/mrdt.js]: Is it ok if I leave it as is and we can cover it as part of a larger code style discussion at some point?

Comment (dianabarsan) [reply webapp/src/js/enketo/widgets/mrdt.js]: Sure! No problem.

**Pull request merged**